catalog-rest-service/src/test/java/org/openmetadata/catalog/resources/lineage/LineageResourceTest.java

```diff
@@ -64,6 +81,54 @@ public static void setup(TestInfo test) throws IOException, URISyntaxException {
     }
   }
 
+  @Test
+  void put_delete_lineage_withAuthorizer(TestInfo test) throws HttpResponseException {
```

Comment:
More lines of code for unit tests than actual code for this feature.

Also keeping the entities local to this test as @amiorin suggested to avoid globals in #2256